proto/regen/group/v1alpha1/types.proto

```diff
@@ -121,6 +121,11 @@ message GroupAccountInfo {
 
     // decision_policy specifies the group account's decision policy.
     google.protobuf.Any decision_policy = 6 [(cosmos_proto.accepts_interface) = "DecisionPolicy"];
+
+    // path is the "derivation" path of the group account ModuleID,
+    // which is needed to derive the group root module key and execute proposals.
+    bytes path = 7;
```

Comment:
Let's use `key` or `derivation_key`, rather than path. 
Path is misleading, and we don't do a multi derivation here.

Reply:
yes, generally this PR will need to be updated to be in accordance with https://github.com/regen-network/regen-ledger/pull/313